front-end/src/Components/SnackNewForm.js

```diff
@@ -0,0 +1,129 @@
+import { useState } from "react";
+import axios from "axios";
+
+import { useNavigate } from "react-router-dom";
+import "./SnackNewForm.css";
+
+const API_URL = process.env.REACT_APP_API_URL;
+
+function SnackNewForm() {
+  const [snack, setSnack] = useState({
+    name: "",
+    fiber: 0,
+    protein: 0,
+    added_sugar: 0,
+    is_healthy: false,
+    url: "",
+  });
+
+  const navigate = useNavigate();
+
+  const addSnack = () => {
+    axios
+      .post(`${API_URL}/snacks`, snack)
+      .then((res) => {
+        navigate(`/snacks`);
+      })
+      .catch((error) => console.error(error));
+  };
+
+  const handleTextChange = (e) => {
+    setSnack({ ...snack, [e.target.id]: e.target.value });
+  };
+
+  const handleNumberChange = (e) => {
+    setSnack({ ...snack, [e.target.id]: Number(e.target.value) });
+  };
+
+  const handleUrlChange = (e) => {
+    setSnack({ ...snack, [e.target.id]: e.target.value });
+  };
+
+  const handleSubmit = (e) => {
+    e.preventDefault();
+    addSnack();
+  };
+
+  return (
+    <div className="new">
+      <br />
+      <br />
+      <br />
+      <br />
```

Comment:
we could do much better in styling by using CSS than chaining <br/ > tags - consider adding some margin bottom spacing